spec/lib/bib_item_set_spec.rb

```diff
@@ -21,6 +21,12 @@
       expect(bib_item_set.filter_missing_and_lost.size).to eq 2
     end
 
+    describe '#filter_techserv_unassigned_and_intref' do
+      it 'filters specific locaation codes' do
+      expect(bib_item_set.filter_techserv_unassigned_and_intref.size).to eq 3
```

Comment:
@sensei100 there is a bit of ambiguity with this test.  On line 16 above I see that bib_items_set.size is 3 . so I'm not sure what is getting filtered since we already started with 3 items?

* Add an item that will get filtered to your fixture.
* Update your test with new numbers... probably 4, 2, 3?

Also it looks like from the current implementation of `filter_techserv_unassigned_and_intref` that you do not want the original item to get mutated (as per clone/dup).  So perhaps adding a test to very mutation did not take place is good so that you guard against a future change that causes that.